clients/drcachesim/tracer/tracer.cpp

```diff
@@ -245,6 +264,7 @@ event_bb_setup(void *drbbdup_ctx, void *drcontext, void *tag, instrlist_t *bb,
     DR_ASSERT(enable_dups != NULL && enable_dynamic_handling != NULL);
     if (bbdup_duplication_enabled()) {
         *enable_dups = true;
+        /* Make sure to update opts.non_default_case_limit if adding an encoding here. */
```

Comment:
style nit: Prefer // comment for full-line comment in C++ code.

Reply:
Changed.